pkgs/tools/misc/cyclonedx-python/default.nix

```diff
@@ -1,65 +1,43 @@
-{ lib
-, fetchFromGitHub
-, python3
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  nix-update-script,
```

Comment:
I don't think that we need the update-script as the update process is straight-forward and `nix-update` will work.